specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/transfers.json

```diff
@@ -0,0 +1,1194 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-04-01",
+    "title": "BillingManagementClient",
+    "description": "Billing client provides access to billing resources for Azure subscriptions."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingProfiles/{billingProfileName}/invoiceSections/{invoiceSectionName}/transfers/{transferName}": {
+      "get": {
+        "tags": [
+          "Transfers"
+        ],
+        "description": "Gets a transfer request by ID. The operation is supported only for billing accounts with agreement type Microsoft Customer Agreement.",
+        "operationId": "Transfers_Get",
+        "x-ms-examples": {
+          "TransferGet": {
+            "$ref": "./examples/GetTransfer.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./types.json#/parameters/billingAccountName"
+          },
+          {
+            "$ref": "./types.json#/parameters/billingProfileName"
+          },
+          {
+            "$ref": "./types.json#/parameters/invoiceSectionName"
+          },
+          {
+            "$ref": "#/parameters/transferNameParameter"
+          },
+          {
+            "$ref": "./types.json#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Details of transfer.",
+            "schema": {
+              "$ref": "#/definitions/TransferDetails"
+            }
+          },
+          "default": {
+            "description": "Unexpected error.",
+            "schema": {
+              "$ref": "./types.json#/definitions/ArmErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
```

Comment:
This should likely be a POST right? PUT is used to create a resource and the PUT request body should match the GET response body. 

This looks more like an action (with an InitiateTransfer as the request body) and a TransferDetails (non-resource) as the response body).